homeassistant/components/openweathermap/strings.json

```diff
@@ -0,0 +1,35 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "OpenWeatherMap integration for these coordinates is already configured."
+        },
+        "error": {
+            "auth": "API key is not correct.",
```

Comment:
Please use as many standard references as possible. Here are the available references:

https://github.com/home-assistant/core/blob/3565fec0051f78309ee1398987a0998aea6c62e7/homeassistant/strings.json#L20-L57

Here's how to use a reference:
https://github.com/home-assistant/core/blob/3565fec0051f78309ee1398987a0998aea6c62e7/homeassistant/components/adguard/strings.json#L7

Reply:
already fixed